user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Cassandra_general/Cassandra_updating.md

```diff
@@ -32,6 +32,10 @@ To update the Cassandra version:
 1. Ensure you have a **full backup** of the Cassandra database.
 1. Download the latest Cassandra *3.11.X* binaries from the [official website](https://cassandra.apache.org/_/download.html) and *extract* the archive.
 1. Stop the DataMiner agent.
+1. Push all data from memory to disk and stop processing new requests. Go to the *C:\Program Files\Cassandra\bin* folder and run the following command:
```

Comment:
@SkylineCookieMonster Do you mean "To push all data from memory to disk and stop processing new requests, go to the *C:\Program Files\Cassandra\bin* folder and run the following command:"? 
If not, could you separate these steps and explain how you push all data from memory to disk and stop processing new requests?

Reply:
@MariekeGO Indeed the 'nodetool drain' command will stop all incoming requests to be handled and will offload all data that is still in memory to ssTables (on disk).